plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly

```diff
@@ -1,6 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:st="jelly:stapler">
-  <l:layout norefresh="true" type="one-column" title="${%Configuration as Code}">
+  <l:layout norefresh="true" type="one-column" title="${%Configuration as Code}" permission="${app.SYSTEM_READ}">
```

Comment:
there's a `permissions` field that can use an array of permissions see for something similar https://github.com/jenkinsci/jenkins/blob/677fbb2a81fa8e9a0bc37ad5eca56565612e1f2e/core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly#L39